frontend/internal-packages/agent/src/db-agent/prompt.ts

```diff
@@ -67,10 +71,12 @@ The current schema structure will be provided:
     "value": {{
       "name": "posts",
       "columns": {{
-        "id": {{"name": "id", "type": "uuid", "notNull": true, "default": "gen_random_uuid()", "comment": "Primary key", "check": null}},
-        "user_id": {{"name": "user_id", "type": "uuid", "notNull": true, "default": null, "comment": "References users", "check": null}}
+        "id": {{"name": "id", "type": "uuid", "notNull": true, "default": "gen_random_uuid()", "comment": "Unique identifier for each post", "check": null}},
+        "user_id": {{"name": "user_id", "type": "uuid", "notNull": true, "default": null, "comment": "Author of the post, links to users.id", "check": null}},
+        "title": {{"name": "title", "type": "text", "notNull": true, "default": null, "comment": "Post headline displayed in listings and search results", "check": null}},
+        "content": {{"name": "content", "type": "text", "notNull": false, "default": null, "comment": "Main body content in markdown or HTML format", "check": null}}
```

Comment:
It was unnatural that the posts table example didn't have a column for content until now. Thank you!
If you want to compress the information, you can delete the title this time.

```suggestion
        "content": {{"name": "content", "type": "text", "notNull": false, "default": null, "comment": "Main body content in markdown or HTML format", "check": null}}
```